### PR TITLE
Adding functions to just calculate point estimate without SE. 

### PR DESCRIPTION
**Pull request by ayushpatnaikgit**
Had to significantly reorganise the tests. Each testset was loading datasets and creating a survey design, on many occasions incorrectly. I have loaded datasets and made survey designs right in runtests.jl and used them everywhere.

### CODE REVIEW

**Comment by ayushpatnaikgit** [comment]
I think it can go straight into main, and we don't need to push it to `novar`.

File: test/runtests.jl [hunk 1]
```diff
@@ -2,9 +2,26 @@ using Survey
...
+apistrat = load_data("apistrat") # Load API dataset
+strat = SurveyDesign(apistrat, strata = :stype, weights = :pw) # Create SurveyDesign
+strat_boot = strat |> bootweights # Create replicate design
+
+# One-stage cluster sample
+apiclus1 = load_data("apiclus1") # Load API dataset
+apiclus1[!, :pw] = fill(757/15,(size(apiclus1,1),)) # Correct api mistake for pw column
+dclus1 = SurveyDesign(apiclus1; clusters =  :dnum, weights = :pw) # Create SurveyDesign
+dclus1_boot = dclus1 |> bootweights # Create replicate design
+
```
Comment (smishr) [inline test/runtests.jl:22]: so this is where to keep global data objects?

**Review by smishr — APPROVED** [review]
okay looks good to merge?

File: test/runtests.jl (hunk 1 above)
Comment (ayushpatnaikgit) [reply test/runtests.jl:22]: Yes, once and for all in runtests.jl

**Pull request merged**